Makefile

```diff
@@ -7,6 +7,7 @@ install:
 
 test:
 	npm test
+	npm run nsp
```

Comment:
another option would be changing the `.travis.yml` to include this and linting as well.